docs/bnb_smart_chain/staking/index.md

```diff
@@ -0,0 +1,191 @@
+
+## Validator Operations
+
+Validators are nodes running BNB Smart Chain software, participating in the consensus process.
+They require a minimum BNB stake at their validator address and can receive delegations from other BNB holders.
+Validators earn rewards from transaction fees and share a part of these rewards with their delegators.
+
+### Create Validator
+
+To create a validator, a BNB holder needs to send a `CreateValidator` transaction to the `StakeHub` contract,
+which is a system contract and the address is `0x0000000000000000000000000000000000002002`,
+with minimum amount of BNB that the validator needs to stake to their own validator
+address (2000 BNB), specifying the following information:
+
+- **Operator address**: The address of the validator, which will receive the staking credit and the rewards.
+- **Consensus address**: The consensus address of the validator's node.
+- **Vote Address**: The address for participating fast finality voting.
+- **BLS Proof**: A BLS signature to prove that the validator owns the vote address.
+- **Commission**: The commission rate defines the percentage of the rewards that the validator will keep for themselves,
+  and the rest will be distributed to the delegators. It also contains the max commission rate, the max change rate during
+  a predefined timespan for validator to set.
+- **Description**: The optional information about the validator, such as moniker, identify, website, etc.
+
+The `CreateValidator` transaction will deduct the minimum self-delegation amount from the validator address and issue
+the corresponding staking credit to the validator. The validator will then join the standby validator set, and wait for
+the next validator set update to see if they can enter the active validator set.
+
+### Edit Validator
+
+A validator can edit their validator information by sending  `EditConsensusAddress`, `EditCommissionRate`,
+`EditDescription`, `EditVoteAddress` transactions to the `StakeHub` contract, specifying the following information
+accordingly:
+
+- **New consensus address**: The new consensus address of the validator's node.
+- **New commission**: The new percentage of the rewards that the validator will keep for themselves, which can
+  only be increased within a maximum change rate limit.
+- **New description**: The new information about the validator, such as moniker, identify, website, etc.
+- **New vote address**: The new vote address for participating fast finality.
+
+These transactions will update the validator information on the BNB smart chain, and the changes will take
+effect immediately. However, the new commission rate will only apply to the rewards earned after the transaction, and
+the previous rewards will be distributed according to the previous commission rate.
+
+## Delegator Operations
+
+Delegators are BNB holders who stake their BNB with a validator, sharing rewards and risks.
```

Comment:
delete `and risks`, currently there is no risk.

Reply:
fixed